account_asset_management_stock_lot/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Assets Management Stock Lot",
+    "version": "16.0.1.2.5",
```

Comment:
```suggestion
    "version": "16.0.1.0.0",
```
`0.0` in the first version